cli/src/commands/schema/edit.ts

```diff
@@ -33,10 +34,35 @@ type EditableTable = Table & {
   columns: EditableColumn[];
 };
 
+type ColumnEditState = {
+  initial: {
+    name: string;
+    type: string;
+    link: string | undefined;
+    notNull: string;
+    defaultValue: string;
+    unique: string;
+    description: string | undefined;
+  };
+  values: {
+    name?: string;
+    type?: string;
+    link?: string;
+    notNull?: string;
+    defaultValue?: string;
+    unique?: string;
+    description?: string;
+  };
+};
+
 const types = ['string', 'int', 'float', 'bool', 'text', 'multiple', 'link', 'email', 'datetime'];
 const typesList = types.join(', ');
 const identifier = /^[a-zA-Z0-9-_~]+$/;
 
+const uniqueUnsupportedTypes: string[] = ['text', 'multiple'];
+const defaultValueUnsupportedTypes: string[] = ['multiple', 'link'];
+const notNullUnsupportedTypes: string[] = defaultValueUnsupportedTypes;
```

Comment:
[types] Isn't it the same type as not adding explicitly string[]?

Reply:
Thanks copy paste mistake!